pom.xml

```diff
@@ -310,6 +310,32 @@
                     <mainClass>com.authlete.common.util.CLI</mainClass>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.3.1</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>10.10.0</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.example</groupId>
```

Comment:
Note: remember to release authlete-checkstyle to a public registry before merging, and changing this.